configure.ac

```diff
@@ -512,23 +512,29 @@ EXTRA_WARNINGS=""
 
 WARNLIST="
 	all
+	extra
+	unused
 	shadow
 	missing-prototypes
 	missing-declarations
+	suggest-attribute=noreturn
```

Comment:
This flags functions that the compiler thinks never return as good candidates for the noreturn attribute. When the noreturn attribute is used in the headerfile / function declaration, it allows the compiler to optimize the call-site of the function in other translation units, such as not bothering to fill out the full framework for a function call, since the called function won't ever return back to it.

If this never triggers, that's great. Otherwise it'll tell of places where we can capture additional optimization.